extensions/amp-analytics/0.1/requests.js

```diff
@@ -157,32 +151,19 @@ export class RequestHandler {
       });
     }
 
-    const extraUrlParamsPromise = getExtraUrlParams(
-        this.variableService_, configParams, triggerParams, expansionOption)
-        .then(expandExtraUrlParams => {
-          // Construct the extraUrlParamsString: Remove null param and encode
-          // component
-          const expandedExtraUrlParamsStr = getExtraUrlParamsString(
-              this.variableService_, expandExtraUrlParams);
-          return this.urlReplacementService_.expandUrlAsync(
```

Comment:
`this.urlReplacementService_.expandUrlAsync` is moved into `expandExtraUrlParams`